modules/@apostrophecms/piece-type/index.js

```diff
@@ -1206,6 +1206,43 @@ module.exports = {
 
           console.log(`Done unlocalizing module ${self.name}`);
         }
+      },
+
+      touch: {
+        usage: 'Invoke this task to touch (update without any change) all docs of this type.',
+        async task(argv) {
+          const req = self.apos.task.getAdminReq();
+          let errCount = 0;
+          let count = 0;
+
+          // We have 30 minutes (by default) for each iterration.
+          // https://www.mongodb.com/docs/manual/reference/method/cursor.noCursorTimeout/#session-idle-timeout-overrides-nocursortimeout
+          const cursor = (await self.find(req, {})
```

Comment:
Need a `try/finally` block here to ensure `cursor.close` gets called no matter what so we don't leak one. (Actually I'm not sure if it's OK to call it if the cursor is successfully exhausted, you'll want to check on that.)

OK we won't really leak one in a task because the lifetime is limited, but we want to get in the habit of cleaning them up.